cache-extension-demo/example-function/index.js

```diff
@@ -6,7 +6,7 @@ exports.handler = function(event, context, callback) {
     const options = {
         hostname: 'localhost',
         port: 3000,
-        path: '/dynamodb/DynamoDbTable-pKey1-sKey1',
+        path: '/dynamodb?name=DynamoDbTable-pKey1-sKey1',
```

Comment:
Do the hyphens take into account table names that include hyphens? What about keys that include hyphens?